gettsim/transfers/grunds_im_alter.py

```diff
@@ -0,0 +1,445 @@
+import pandas as pd
+
+from gettsim.piecewise_functions import piecewise_polynomial
+from gettsim.typing import BoolSeries
+from gettsim.typing import FloatSeries
+from gettsim.typing import IntSeries
+
+
+def grunds_im_alter_m_hh(
+    grunds_im_alter_m_minus_eink_hh: FloatSeries,
+    alle_erwachsene_sind_rentner_hh: BoolSeries,
+    vermögen_hh: FloatSeries,
+    grunds_im_alter_vermög_freib_hh: FloatSeries,
+) -> FloatSeries:
+    """Calculate Grundsicherung im Alter on household level.
+
+    # ToDo: There is no check for Wohngeld included as Wohngeld is
+    # ToDo: currently not implemented for retirees.
+
+    # ToDo: Grundsicherung im Alter is only paid if all adults in the household
+    # ToDo: are retired. Other households get ALG 2 (This is a simplification by
+    # ToDo: GETTSIM)
+
+    Parameters
+    ----------
+    grunds_im_alter_m_minus_eink_hh
+        See :func:`grunds_im_alter_m_minus_eink_hh`.
+    alle_erwachsene_sind_rentner_hh
+        See :func:`alle_erwachsene_sind_rentner_hh`.
+    vermögen_hh
+        See basic input variable :ref:`vermögen_hh`.
+    grunds_im_alter_vermög_freib_hh
+        See :func:`grunds_im_alter_vermög_freib_hh`.
+    Returns
+    -------
+
+    """
+    out = grunds_im_alter_m_minus_eink_hh.copy()
+
+    # Wealth check
+    out.loc[vermögen_hh >= grunds_im_alter_vermög_freib_hh] = 0
+
+    # Only pay Grundsicherung im Alter if all adults are retired (see docstring)
+    out.loc[~alle_erwachsene_sind_rentner_hh] = 0
+    return out
+
+
+def grunds_im_alter_m_minus_eink_hh(
```

Comment:
I find the name confusing, needed to check with `grunds_im_alter_m_hh` several times what is done here and there. I don't really think this requires a separate function; why not put income and wealth checks *(plus the "everybody retired" check, which is doomed anyhow)* into one function `grunds_im_alter_m_hh`  ?

If it stays, the name should be `grunds_im_alter_minus_eink_m_hh` for a start, but probably would see changes.